# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-pseudo/text-selection.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/css/css-pseudo/text-selection.html.ini
--- a/testing/web-platform/meta/css/css-pseudo/text-selection.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/css/css-pseudo/text-selection.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,17 +1,17 @@
 [text-selection.html]
   [Selection ending in ::marker]
     expected:
+      if debug and (os == "linux") and webrender and not fission and not swgl: ["FAIL", "PASS"]
+      if debug and (os == "linux") and webrender and not fission and swgl: ["PASS", "FAIL"]
+      if debug and (os == "linux") and webrender and fission: ["PASS", "FAIL"]
+      if debug and (os == "linux") and not webrender: ["PASS", "FAIL"]
+      if debug and (os == "mac"): ["PASS", "FAIL"]
+
+  [Selection ending in ::before-marker]
+    expected:
       if debug and (os == "linux") and webrender and not fission and swgl: ["PASS", "FAIL"]
       if debug and (os == "linux") and webrender and not fission and not swgl: ["FAIL", "PASS"]
       if debug and (os == "linux") and webrender and fission: ["PASS", "FAIL"]
       if debug and (os == "linux") and not webrender: ["PASS", "FAIL"]
       if debug and (os == "mac"): ["PASS", "FAIL"]
 
-  [Selection ending in ::before-marker]
-    expected:
-      if debug and (os == "linux") and webrender and not fission and not swgl: ["FAIL", "PASS"]
-      if debug and (os == "linux") and webrender and not fission and swgl: ["PASS", "FAIL"]
-      if debug and (os == "linux") and webrender and fission: ["PASS", "FAIL"]
-      if debug and (os == "linux") and not webrender: ["PASS", "FAIL"]
-      if debug and (os == "mac"): ["PASS", "FAIL"]
-